src/main/java/seedu/exercise/logic/commands/AddCommand.java

```diff
@@ -18,23 +18,22 @@
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a exercise to the exercise list. "
         + "Parameters: "
-        + PREFIX_NAME + "NAME "
-        + PREFIX_DATE + "PHONE "
-        + PREFIX_CALORIES + "EMAIL "
-        + PREFIX_QUANTITY + "ADDRESS "
+        + PREFIX_NAME + "EXERCISE NAME "
+        + PREFIX_DATE + "DATE "
+        + PREFIX_CALORIES + "CALORIES "
+        + PREFIX_QUANTITY + "QUANTITY "
         + "[" + PREFIX_MUSCLE + "TAG]...\n"
         + "Example: " + COMMAND_WORD + " "
-        + PREFIX_NAME + "John Doe "
-        + PREFIX_DATE + "98765432 "
-        + PREFIX_CALORIES + "johnd@example.com "
-        + PREFIX_QUANTITY + "311, Clementi Ave 2, #02-25 "
-        + PREFIX_MUSCLE + "friends "
-        + PREFIX_MUSCLE + "owesMoney";
+        + PREFIX_NAME + "Run "
+        + PREFIX_DATE + "22/09/2019 "
+        + PREFIX_CALORIES + "1500 "
+        + PREFIX_QUANTITY + "2.4 "
+        + PREFIX_MUSCLE + "Leg ";
```

Comment:
A whitespace not required at the end of the "Leg" string.

Reply:
done